cider-velox/src/CiderOperator.h

```diff
@@ -62,11 +62,11 @@ class CiderOperator : public exec::Operator {
   std::chrono::microseconds convertorInternalCounter;
 
   // these properties are used for join with w/o agg case
-  std::optional<CiderBatch> buildData_;
+  std::optional<std::shared_ptr<CiderBatch>> buildData_;
```

Comment:
maybe we don't need this member , since we move into cidercompile module?

Reply:
Yes, deleted the buildData_.